documentation/book/proc-deleting-a-topic.adoc

```diff
@@ -0,0 +1,38 @@
+[id='deleting-a-topic_{context}']
+= Deleting a topic
+
+This procedure describes how to delete a Kafka topic using a `KafkaTopic` {ProductPlatformName} resource.
+
+.Prerequisites
+
+* A running Kafka cluster.
+* A running Topic Operator, either deployed by the Cluster Operator using the `Kafka.spec.topicOperator` property, or deployed standalone.
```

Comment:
Can standalone be done in the product?

Reply:
@scholzj can you comment on that, please?

Reply:
Feel free to merge if it you deploy standalone in the product, if not, then we'll want some sort of 'if' around that clause.

Reply:
I do not see any reason why standalone Topic Operator should not be supported in AMQ Streams. It isn't any new functionality, just different deployment. However I think it is super low prio because only very few would be interested. So adding the docs for it should be low prio IMHO.